ARCs/arc-0029.md

```diff
@@ -0,0 +1,101 @@
+---
+arc: 0029
+title: Algorand Standard Asset Parameters Conventions for Soul Bound NFTs
+author:
+- Lukas Kotol (@hathoriel)
+  discussions-to: https://github.com/algorandfoundation/ARCs/pull/117
+  status: Draft
+  created: 2022-08-06
+---
+
+## Summary
+
+This document introduces conventions for the parameters of Algorand Standard Assets (ASAs) for Soul Bound non-fungible tokens (NFTs).
+
+## Abstract
+
+A "Soul Bound NFT" is an Algorand Standard Asset (ASA) defined by configuration of parameters preventing
+them to be transferred from an original address.
+
+## Motivation
+
+In the popular MMORPG World of Warcraft, its game designers intentionally took some items out of the world’s
+auction house market system to prevent them from having a publicly-discovered price and limit their accessibility.
+
+Vanilla WoW’s “Thunderfury, Blessed Blade of the Windseeker” was one such legendary item, and it
+required a forty-person raid, among other sub-tasks, to slay the firelord “Ragnaros” to gain
+the “Essence of the Firelord,” a material needed to craft the sword once.
+
+Upon voluntary pickup and craft, the sword permanently binds to a character’s “soul,” making
+it impossible to trade, sell or even swap it between a player’s characters.
+
+In other words, “Thunderfury”’s price was the aggregate of all social costs
+related to completing the difficult quest line with friends and guild members.
+Other players spotting Thunderfuries could be sure
+their owner had slain “Ragnaros,” the blistering firelord.
+
+World of Warcraft players could trash legendary and soulbound items like the Thunderfury to permanently remove
+them from their account.
```

Comment:
I understand this is a submission for the gitcoin bounty https://gitcoin.co/issue/29133 and so it uses the term Soulbound. This paragraph is trying to explain the terminology "Soulbound", which is needed to give context. However, the highlighted part is a blatant copypaste from [EIP-4973](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4973.md#motivation) to this proposed ARC. I'm sure the concept could be explained with other examples and wordings instead of a direct copy of words.

Reply:
OK, thanks for the review! I will try to reformulate it better.

Reply:
Hi @fionnachan, I think I have fixed this issue with new wording and examples.

Reply:
Thanks @fionnachan for flagging this!
Since EIP are CC0, in theory, I guess we could use directly without citation.
However, I would strongly recommend that EIP-4973 is credited if anything is taken from there (even actually if not strictly copy-pasted).